requirements-optional.txt

```
@@ -1,7 +1,7 @@
 open3d==0.10.0.0; platform_machine != "armv6l" and platform_machine != "armv7l" and python_version < "3.9" and platform_machine != "aarch64"
-ffmpy3==0.2.4
-pyusb==1.2.1
-sentry-sdk==1.5.1
+ffmpy3
+pyusb
+sentry-sdk
```

```suggestion
ffmpy3>=0.2.4
pyusb>=1.2.1
sentry-sdk>=1.5.1
```

+1 (though it'd be better to have a min working version specified IMO)

But CC: @daniilpastukhov @Erol444  
And side question, why is `pyusb` needed?

---

`pyusb` is used to make a system report (in depthai_demo.py)

---

@PetrNovota 

Actually - I misread these as being SDK deps - in case of these being end application dependencies, should rather be pinned